.github/workflows/integrationtests.yml

```diff
@@ -58,12 +62,12 @@ jobs:
             fi
           done
           echo "Error: Server did not start within 2 minutes."
-          exit 1       
```

Comment:
Lets not have this change which is unnecessary

Reply:
I have removed the unnecessary change